README.md

```diff
@@ -1,2 +1,103 @@
+
+## Clone repository
+
+First you need to clone the repository for the new Task. After cloning move to that folder.
+
+## Install the template
+
+You can install the template by following the command.
+
+`dotnet new -i frendstasktemplate --nuget-source https://pkgs.dev.azure.com/frends-platform/frends-tasks/_packaging/main/nuget/v3/index.json` 
+
+## Create a new Task
+
+You can create a new Task by running the following command in the Task's repository folder.
+
+`dotnet new frendstasktemplate --name namespaceForTask --className classNameForTask --taskName TaskName --workflows TaskName --allow-scripts yes`
+
+## Get help using the template
+
+`dotnet new frendstasktemplate -h`
+
+Example output:
+
+```
+Usage: new [options]
+
+Options:
+  -h, --help          Displays help for this command.
+  -l, --list          Lists templates containing the specified name. If no name is specified, lists all templates.
+  -n, --name          The name for the output being created. If no name is specified, the name of the current directory is used.
+  -o, --output        Location to place the generated output.
+  -i, --install       Installs a source or a template pack.
+  -u, --uninstall     Uninstalls a source or a template pack.
+  --nuget-source      Specifies a NuGet source to use during install.
+  --type              Filters templates based on available types. Predefined values are "project", "item" or "other".
+  --dry-run           Displays a summary of what would happen if the given command line were run if it would result in a template creation.
+  --force             Forces content to be generated even if it would change existing files.
+  -lang, --language   Filters templates based on language and specifies the language of the template to create.
+  --update-check      Check the currently installed template packs for updates.
+  --update-apply      Check the currently installed template packs for update, and install the updates.
+
+
+Official Task template (C#)
+Author: Frends
+Options:
+  -t|--taskName
+                            string - Optional
+                            Default: Task
+
+  -c|--className
+                            string - Optional
+                            Default: Template
+
+  -au|--authors
+                            string - Optional
+                            Default: Frends
+
+  -li|--license
+                            string - Optional
+                            Default: MIT
+
+```
+
+## Update the template
+
+`dotnet new frendstasktemplate --update-apply `
+
+## Uninstall the template
+
+`dotnet new -u frendstasktemplate`
+
+## Building this repository
+
+`dotnet pack`
+
+This command will build the project and create a NuGet package in .\bin\Debug 
+
+To install this template from locally created NuGet package use
+
+`dotnet new -i <ABSOLUTE_PATH_TO_NUPKG_FILE>`
+
+
```

Comment:
Somewhere in the README we should have a note about template development. Like "to develop this you do the modifications and then run `dotnet install .` to test this".